Cargo.lock

```diff
@@ -2324,9 +2326,9 @@ dependencies = [
 
 [[package]]
 name = "quote"
-version = "1.0.23"
+version = "1.0.25"
```

Comment:
@dtolnay it looks like this release might have had an unintentional breaking change? `diesel_derive` does not seem to compile anymore. should this be fixed in `quote` or is `diesel_derive` using it wrong? 😅

Reply:
appears to be fixed with 1.0.26